python/src/sdf/pyWorld.cc

```diff
@@ -163,6 +163,13 @@ void defineWorld(pybind11::object module)
           "Set the atmosphere model associated with this world.")
      .def("physics_count", &sdf::World::PhysicsCount,
           "Get the number of physics profiles.")
+     .def("physics_by_index",
+          pybind11::overload_cast<uint64_t>(
+            &sdf::World::PhysicsByIndex),
+          pybind11::return_value_policy::reference_internal,
+          "Get a mutable physics profile based on an index.")
+     .def("physics_name_exists", &sdf::World::PhysicsNameExists,
```

Comment:
Binding from `World::PhysicsDefault` is missing

Reply:
https://github.com/gazebosim/sdformat/pull/995/commits/303aa17e5bffb05a5fe8a79456e84470aad866fe